homeassistant/components/zha/core/cluster_handlers/__init__.py

```diff
@@ -323,24 +325,38 @@ def _configure_reporting_status(
                 self.name,
                 res,
             )
+            # 2.5.8.1.3 Status Field
+            # The status field specifies the status of the Configure Reporting operation attempted on this attribute, as de- tailed in 2.5.7.3.
```

Comment:
```suggestion
            # The status field specifies the status of the Configure Reporting operation attempted on this attribute, as detailed in 2.5.7.3.
```
This got lost with the changes.

Reply:
that's what I get for not paying attention :D thanks again.